blockio-uring.cabal

```diff
@@ -41,7 +41,7 @@ library
     System.IO.BlockIO.URingFFI
 
   build-depends:
-    , base       >=4.12  && <4.20
+    , base       >=4.12  && <4.21
```

Comment:
The lower bound still allows GHC-8.6, e.g.

Reply:
Yes, thanks, fixed.